### ISSUE TITLE
Drop Zenhub Release Report

### ISSUE THREAD

**Issue by emilyrohrbough** [cypress-io/cypress#25156]
@emilyrohrbough commented on [Tue Sep 20 2022](https://github.com/cypress-io/cypress-internal/issues/483)

After successfully release using Git History Script, update PULL_REQUEST requirement to tag with Zenhub Release version & update release steps to close/create/move issues from/to Zenhub release reports.

Update Release Steps to remove reference.

**Comment by emilyrohrbough** [comment]
Blocked by https://github.com/cypress-io/cypress/issues/25155.

**Comment by emilyrohrbough** [comment]
Closing as this work has been “done". The PR to remove the logic from the satisfied this work however, the webhook itself is just broken and I wanted to fix it. I created a new issue to encompass that work since it is different and outside of the scope of this task. See: https://github.com/cypress-io/release-automations/issues/80
